src/assets/sql/stored-procedure/getAddressBookingStatisticByConditions.sql

```diff
@@ -0,0 +1,50 @@
+DROP PROCEDURE IF EXISTS `getAddressBookingStatisticByConditions`;
+CREATE PROCEDURE `getAddressBookingStatisticByConditions`(IN `idServiceIntegrated` VARCHAR(200), IN `cityId` INT(11), IN `startDate` VARCHAR(200), IN `endDate` VARCHAR(200), IN `gender` VARCHAR(200), IN `first_time` VARCHAR(200), IN `bhyt` VARCHAR(200))
+BEGIN
+    IF startDate IS NULL OR startDate = '' THEN
```

Comment:
please remove check empty string here, then expect params value are either the NULL or a valid value
In the controller, you are to check if the value is empty, to push null  as param value to the procedure

Reply:
updated